pkgs/applications/gis/qgis/unwrapped-ltr.nix

```diff
@@ -6,8 +6,8 @@
 , wrapGAppsHook3
 , wrapQtAppsHook
 
-, withGrass ? true
-, withWebKit ? false
+, withGrass
+, withWebKit
```

Comment:
That's a detail, but we could have kept the default values there, sometimes I directly use the unwrapped version so that it's easier to override other stuff.

Thanks for this PR anyway!

Reply:
I didn't want to duplicate default values, and they where needed in in wrapped package (otherwise it didn't evaluate).

Reply:
Yes I meant to duplicate the default values so that overwritting the unwrapped package stays easy. Not a big deal, we need to find a better solution for all this anyway...